helm/hpcc/values.schema.json

```diff
@@ -598,8 +598,13 @@
     },
     "ldap": {
       "type": "object",
+      "required": [ "ldapAddress" ],
       "additionalProperties": { "type": ["integer", "string", "boolean"] },
       "properties": {
+        "ldapAddress": {
+          "type": "string",
+          "description": "LDAP Server IP address(es), comma separated"
```

Comment:
trivial: wouldn't hold up merging now for this, but should avoid using IP's in general, so should probably be changed to say "LDAP Server hostname(s), comma separated"